src/main/java/com/google/collegeplanner/data/Course.java

```diff
@@ -84,31 +80,52 @@
   private String creditGrantedFor;
 
   public Course(String courseId, String name, String semester, int credits, String departmentId,
-      String description, String[] gradingMethod, String corequisites, String prerequisites,
-      String restrictions, String additionalInfo, String creditGrantedFor) {
+      String description, String corequisites, String prerequisites, String restrictions,
+      String additionalInfo, String creditGrantedFor) {
     this.courseId = courseId;
```

Comment:
since courseId is being used as a key now, do we need to normalize it (uppercase, lowercase, etc) before putting it in? Otherwise MATH101 and Math101 will be two separate objects.